routing/pathfind.go

```diff
@@ -1112,3 +1108,54 @@ func getProbabilityBasedDist(weight int64, probability float64,
 
 	return int64(dist)
 }
+
+// lastHopPayloadSize calculates the payload size of the final hop in a route.
+// It depends on the tlv types which are present and also whether the hop is
+// part of a blinded route or not.
+func lastHopPayloadSize(r *RestrictParams, finalHtlcExpiry int32,
+	amount lnwire.MilliSatoshi, legacy bool) uint64 {
+
+	if r.BlindedPayment != nil {
+		blindedPath := r.BlindedPayment.BlindedPath.BlindedHops
+		blindedPoint := r.BlindedPayment.BlindedPath.BlindingPoint
+
+		//nolint:lll
```

Comment:
hmm think the contribution guidelines says we should try our best to not exceed this limit, think we can easily fit in by defining a var to hold `blindedPath[len(blindedPath)-1].CipherText`